src/utils/settings.cpp

```diff
@@ -213,6 +214,8 @@ void multipass::Settings::set_aux(const QString& key, QString val) // work with
         throw InvalidSettingsException(key, val, "Invalid flag, try \"true\" or \"false\"");
     else if (key == winterm_key || key == hotkey_key)
         val = mp::platform::interpret_setting(key, val);
+    else if (key == passphrase_key)
+        val = MP_UTILS.generate_scrypt_hash_for(val);
```

Comment:
Well, I avoided this since the whole settings stuff will be changing soon and didn't want to take time writing a test that would have to be changed soon.  But if you think it's worth the time, then I'll do it....reluctantly :wink:

Reply:
No, you're right.